library/src/G4LatticeReader.cc

```diff
@@ -27,13 +27,13 @@
 // 20160802  Use hep_pascal for pressure (Windows compatibility)
 // 20170810  Processing IV scattering matrix terms, allow "/eV" type units
 // 20170821  For deformation potentials, specify eV/cm units; use regex match
-//		for multiple optical IV potentials
+//    for multiple optical IV potentials
 // 20170821  Add transverse sound speed, L->TT fraction
 // 20180815  F. Insulla -- Added IVRateQuad
 // 20181001  M. Kelsey -- Clarify IV rate parameters systematically
 // 20190704  M. Kelsey -- Add 'ivModel' to set default IV function by material
-// 20231017  E. Michaud -- Add 'valleyDir' to set rotation matrix with valley's 
-//		 direction instead of euler angles
+// 20231017  E. Michaud -- Add 'valleyDir' to set rotation matrix with valley's
+//     direction instead of euler angles
```

Comment:
Can you please add a dated comment below this line?  It helps us keep track of why changes were made.  Also, it seems that your text editor collapsed tabs (ASCII 0x09) to four spaces.  If you could fix the lines above this so that the text goes back to being aligned, I'd appreciate it.

Reply:
Yet another thing I can blame Fortran for: getting me in the habit of always translating tabs to spaces. I disabled in my editor for this project, so hopefully this doesn't crop up again. I'll start double-checking before submitting.

Reply:
Thank you very much!  I use Emacs for my editor, so it has language-specific formatting to maintain alignment.  That's been a huge help to me as I've learned Python :-/

I see you also went through and fixed a bunch of my misspellings :-)

Reply:
Having spell check in my editor has been more of a curse than a blessing, I can't ignore the red squiggly lines. It might be time for me to take the dive on Emacs too.